spamgourmet-clone/scripts/install-sg-exim.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#This script expects to be run from the spamgourmet-clone main directory
+echo '========================= spamgourmet server installation start'
+source sg-server-config.sh
+
+cd $SCRIPT_BASE_DIR
+
+function exim4_packages {
+	echo '##########################################################################'
+	echo '### install mandatory packages'
+	echo '##########################################################################'
+	apt-get update; apt-get upgrade; apt-get autoremove
```

Comment:
Most sources have these `RUN` in the Dockerfile directly creating the new layers. Might be worth investigating later the build performance or having a multi-stage build to get a smaller final image size.